front/src/main.html

```diff
@@ -33,34 +33,34 @@
     <div id="body" ng-view autoscroll="true"></div>
     <div class="footer">
         <span class="version">@@version</span>
-        <br><a href="/about">More about Yellow Lab Tools</a><br>
+        <br><a href="<%= baseUrl %>about">More about Yellow Lab Tools</a><br>
         <div class="github"><iframe id="ghbtn" frameborder="0" scrolling="0" width="145px" height="30px"></iframe></div>
     </div>
 
     <!-- build:js /js/all.js -->
```

Comment:
You need to remove the slash here as well, in case someone runs the optimized version of the server.